test/linode_client_test.py

```diff
@@ -285,6 +285,88 @@ def test_get_invoices(self):
         self.assertEqual(invoice.label, "Invoice #123456")
         self.assertEqual(invoice.total, 9.51)
 
+    def test_logins(self):
+        logins = self.client.account.logins()
+        self.assertEqual(len(logins), 1)
+        self.assertEqual(logins[0].id, 1234)
+
+    def test_maintenance(self):
```

Comment:
nit: personally I feel it's clear to set the endpoint url as a local variable, because we use it at least twice in the test case - mock the operation and assert the call_url. We call it as the same variable can reduce the chance of making mistakes. But it's just a style suggestion for test cases, feel free to take it or not!